commands/gymaddress.php

```diff
@@ -61,13 +61,14 @@
     } else if($gym && !empty($info)) {
         debug_log('Adding address for gym with ID: ' . $id);
         debug_log('Gym note: ' . $info);
-        my_query(
+        $stmt = $dbh->prepare(
             "
             UPDATE    gyms
-            SET       address = '{$db->real_escape_string($info)}'
-              WHERE   id = {$id}
+            SET       address = :info
+            WHERE     id = :id
             "
         );
+        $stmt->execute(['info' => $info, 'id' => $id]);
```

Comment:
I suggest/vote to always use bindParam & execute() and not sometimes leaving bindParam aside and just do execute(with => params).
Therefore to be consistent, we should use the same code as in commands/gymname.php:
$stmt->bindParam(':info', $info);
$stmt->bindParam(':id', $id);
$stmt->execute();

Reply:
The guide we talked about actually recommends using execute() parametrization by default and binding only when absolutely necessary due to complicated types. As long as we stick to strings & ints those are not necessary.

So I would vote going in the other direction, stopping binds and instead feeding them in with execute. Just didn't get to all the cases yet due to the sheer amount of queries in the codebase.